examples/getting_started/4_Superdense_coding/4_Superdense_coding.ipynb

```diff
@@ -86,6 +86,7 @@
     "from braket.circuits import Circuit, Gate, Moments\n",
```

Comment:
<p>nit: Outputs are missing for the below cells?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>